_posts/programme/2020-08-19-kickoff.md

```diff
@@ -14,42 +14,26 @@ registration_url: https://docs.google.com/forms/d/e/1FAIpQLSetlGKtfCxxaAjtHEGxaQ
 meeting_url: https://zoom.us/j/95982897830
 last_modified_at: 2020-09-02
 ---
-To kick our series off, we’re delighted to announce that the SORSE Launch
-Event will take place 13:00-15:00 UTC on Wednesday 2nd September. We’re
-pleased to be welcoming two keynote speakers at this event:
+To kick off our series off, we have run the SORSE Launch
```

Comment:
```suggestion
To kick off our series, we ran the SORSE Launch
```

Reply:
Ah, nice catch, the duplicate off. Thanks.

I prefer perfect in this case as it it something that reverberates into the present, but either way is fine.

Reply:
As you like. I just used simple past because you used it down below, too.

Reply:
Nah, it's fine :).